java/src/Ice/src/main/java/com/zeroc/IceInternal/Transceiver.java

```diff
@@ -5,8 +5,24 @@
 package com.zeroc.IceInternal;
 
 public interface Transceiver {
+  /**
+   * Returns the selectable channel used by the thread pool's selector to wait for read or write
+   * readiness.
+   *
+   * @return The selectable channel that will be registered with the thread pool's selector or null
+   *     if the transceiver doesn't use a selectable channel.
+   */
   java.nio.channels.SelectableChannel fd();
 
+  /**
+   * Sets the transceiver ready callback. This method is called by the thread pool to provide a
+   * callback object that the transceiver can use to notify the thread pool's selector when more
+   * data is ready to be read or written by this transceiver.
+   *
+   * @param callback The ready callback provided by the thread pool's selector.
+   * @remark The transceiver implementation typically uses this callback when it buffers data read
```

Comment:
To my knowledge, JavaDoc doesn't support `@remark` tags.
Even though it's a less elegant name, this should probably be:
```suggestion
   * @implNote The transceiver implementation typically uses this callback when it buffers data read
```

Reply:
Good point, I thought it worked. I've integrated the text to the summary instead.